File: app/src/main/java/com/pgshare/studentroomsharingapp/Adapter/UserHelper.java
package com.pgshare.studentroomsharingapp.Adapter;

public class UserHelper {
    private String userId, name, email, phone, gender;

    // Required default constructor
    public UserHelper() {
    }

    public UserHelper(String userId, String name, String email, String phone, String gender) {
        this.userId = userId;
        this.name = name;
        this.email = email;
        this.phone = phone;
        this.gender = gender;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }
}

File: app/src/main/java/com/pgshare/studentroomsharingapp/Adapter/UserHelper.kt
package com.pgshare.studentroomsharingapp.Adapter

class UserHelper {
    var userId: String? = null
    var name: String? = null
    var email: String? = null
    var phone: String? = null
    var gender: String? = null

    // Required default constructor
    constructor()

    constructor(userId: String?, name: String?, email: String?, phone: String?, gender: String?) {
        this.userId = userId
        this.name = name
        this.email = email
        this.phone = phone
        this.gender = gender
    }
}
